File: baekjoon/bronze/level2/test_2711/java/Main.java
package bronze.level2.test_2711.java;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        sc.nextLine();

        for (int i = 0; i < n; i++) {
            String[] input = sc.nextLine().split(" ");
            int index = Integer.parseInt(input[0]);
            String str = input[1];
            String answer = str.substring(0, index - 1) + str.substring(index);
            System.out.println(answer);
        }
    }
}

File: baekjoon/bronze/level2/test_2711/kotlin/Main.kt
package bronze.level2.test_2711.kotlin

fun main() {
    val n = readln().toInt()

    for (i in 1..n) {
        val arr = readln().split(" ")

        for (c in 0..<arr[1].length) {
            if (c != arr[0].toInt() - 1) print(arr[1][c])
        }
        println()
    }
}
